tools/lambda-promtail/lambda-promtail/s3.go

```diff
@@ -252,7 +252,6 @@ func getLabels(record events.S3EventRecord) (map[string]string, error) {
 
 	labels["key"] = record.S3.Object.Key
 	labels["bucket"] = record.S3.Bucket.Name
-	labels["bucket_owner"] = record.S3.Bucket.OwnerIdentity.PrincipalID
```

Comment:
Should we keep the `bucket_owner` label and only remove the verification? I'm not sure if anyone relies on the label being present.

Reply:
As far as I can see it is not used anywhere else, so I guess it is not needed.

Reply:
The `labels` are part part of the payload sent to Loki. So, someone might be missing this label after your change.

Reply:
I have reverted the label.